R-packages/evalcast/R/evaluate.R

```diff
@@ -67,7 +67,7 @@ evaluate_predictions <- function(
   backfill_buffer = 10,
   side_truth = NULL,
   grp_vars = c("forecaster", "forecast_date", "ahead", "geo_value"),
-  geo_type = c("county", "hrr", "msa", "dma", "state")) {
+  geo_type) {
```

Comment:
Can you add the defaults back in here and below? To match `covidcast::covidcast_signal()` use
```r
geo_type = c("county", "hrr", "msa", "dma", "state", "hhs", "nation")
```

Then in the body of the function use:
```r
geo_type <- match.arg(geo_type, c("county", "hrr", "msa", "dma", "state", "hhs", "nation"))
```
The result is that `geo_type` will be the first match, unless there is no match. Then it throws an error.

Reply:
It will also do partial matching, so and argument `geo_type = "coun"` will lead to `geo_type <- "county"` in the function body.

Reply:
You only need to do `geo_type <- match.arg(geo_type)`, no second argument, since it gets its second argument automatically from the function's default arguments

Reply:
You learn something new every day. Now I'll be deleting the second argument all over my hard drive. 😄

Reply:
Done